app/presenters/content_type_resolver.rb

```diff
@@ -0,0 +1,47 @@
+module Presenters
+  class ContentTypeResolver
+    def initialize(content_type)
+      self.content_type = content_type
```

Comment:
Two reasons:

1) I'm `nil` averse. If you misspell an instance variable, it's set to `nil` at runtime, which can lead to some nasty / hard-to-find bugs.

2) The object better encapsulates those fields. Contrived example:

Say we wanted to restrict it such that `content_type` must be a valid MIME type from a whitelist. By using a setter method on the object, we can put that logic in a single place rather than have to check it whenever we set the `content_type` on the object.

I try not to use instance variables anywhere if I can help it (except perhaps memoization).